src/main/java/service/DataReaderImpl.java

```diff
@@ -0,0 +1,43 @@
+package service;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Scanner;
+
+public class DataReaderImpl implements DataReader {
+    double value;
+
+    @Override
+    public Map<Integer, Double> readPricesFromFile(Map<Integer, Double> price, String fileName) {
```

Comment:
Please, remove the price Map from the parameters. It's better to create the map inside the method rather than pass empty one

Reply:
Method readPricesFromFile(String fileName) changed.